deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -393,13 +395,25 @@ async def create_new_job(
 
         if pulse3d_semver >= "0.30.1":
             # Tanner (2/7/23): these params added in earlier versions but there are bugs with using this param in re-analysis prior to 0.30.1
-            params.append("stiffness_factor")
-            params.append("inverted_post_magnet_wells")
+            params += ["stiffness_factor", "inverted_post_magnet_wells"]
         if pulse3d_semver >= "0.30.3":
             params.append("well_groups")
         if pulse3d_semver >= "0.30.5":
             params.append("stim_waveform_format")
 
+        if use_noise_based_peak_finding:
+            params += [
+                "height_factor",
+                "relative_prominence_factor",
+                "noise_prominence_factor",
+                "max_frequency",
+                "valley_search_duration",
+                "upslope_duration",
+                "upslope_noise_allowance_duration",
```

Comment:
I keep meaning to ask about if we can all go over these new params soon.

Reply:
Yeah we should, it's actually not too complicated